packages/core/nuxt-theme-module/theme/components/WishlistSidebar.vue

```diff
@@ -28,16 +28,25 @@
                 :regular-price="$n(wishlistGetters.getItemPrice(product).regular, 'currency')"
                 :special-price="wishlistGetters.getItemPrice(product).special && $n(wishlistGetters.getItemPrice(product).special, 'currency')"
                 :stock="99999"
+                :link="localePath(wishlistGetters.getItemLink(product))"
                 image-width="180"
                 image-height="200"
-                @click:remove="removeItem({ product })"
+                @click:remove="removeFromWishlist({ product })"
                 class="collected-product"
               >
-               <template #configuration>
+                <template #configuration>
                   <div class="collected-product__properties">
                     <SfProperty v-for="(attribute, key) in wishlistGetters.getItemAttributes(product, ['color', 'size'])" :key="key" :name="key" :value="attribute"/>
                   </div>
                 </template>
+                <template #actions>
+                  <SfButton
+                    class="sf-button--text desktop-only"
+                    @click.native="addToCart(product)"
+                  >
+                    Add to cart
```

Comment:
i18n where possible. It will allow customer to change string values globally and allow for translations :)

Reply:
Added